plugins/copr.py

```diff
@@ -35,8 +35,10 @@
 # If that fails, attempt to import the deprecated implementation
 # from the platform module.
 try:
-    from distro import linux_distribution
+    from distro import linux_distribution, os_release_attr
 except ImportError:
+    def os_release_attr():
```

Comment:
this should be:
  def os_release_attr(_):
otherwise, you will get:
TypeError: os_release_attr() takes 0 positional arguments but 1 was given